packages/functions/ios/RNFBFunctions/RNFBFunctionsModule.m

```diff
@@ -54,6 +56,10 @@ @implementation RNFBFunctionsModule
 
     FIRHTTPSCallable *callable = [functions HTTPSCallableWithName:name];
 
+    if(options[@"timeout"]){
```

Comment:
```suggestion
    if (options[@"timeout"]) {
```

looked like there was an auto-quality checker setup, does it check whitespace? This one was inconsistent

I'm only peaking in here because functions timeouts is something I have patch-packaged into v5 (in a brutal/useless/hard-coded-value hack way - not worth sharing) so I'm interested in this in particular :-)

Reply:
also maybe a prettier config and VSCode shared project setting that runs prettier on save (maybe) and as a husky pre-commit lint-staged type deal. Thinking out loud

Reply:
Unfortunately this is Obj-c code - so prettier won't run on this 😅I would love something for Obj-C/Java code though if you have any suggestions

Good spot though as this formatting does need fixing

Reply:
For AnkiDroid we use Codacy service as a github check and it applies our checkstyle and PMD rules. I see Sonar has obj-c support and there is something called oclint https://github.com/oclint/oclint - I think you went with Sonar as a github check? and oclint could run command line as an action. Here's whitespace for objective-c? https://github.com/square/spacecommander

All sorts of stuff to waste time on 🤣